### ISSUE TITLE
importing JSON file in svelte.config.js causes language server crash

### ISSUE THREAD

**Issue by duncanwerner** [sveltejs/language-tools#1979]
### Describe the bug

Importing a JSON file in `svelte.config.js`

```js
import pkg from './package.json' assert { type: 'json' }
console.info({version: pkg.version});
```

results in an error in vscode

```
Error in svelte.config.js

TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".json" for (...path...)/json-import-test/package.json svelte
```

and crashes the language server. VSCode stops restarting it after 5 crashes. 

Svelte builds without errors, seems to work.

### Reproduction

Import json file in `svelte.config.js`

```js
import pkg from './package.json' assert { type: 'json' }
console.info({version: pkg.version});
```

Test repo:
https://github.com/duncanwerner/json-import-test

This was created with npm create svelte@latest, skeleton, typescript strict, eslint enabled.

### Expected behaviour

Language server not crashing

### System Info

- OS: WSL2, Ubuntu 22.04.1 on Windows 11
- IDE: VSCode
- VSCode plugin version: 107.3.0

I went back as far as 105.0.0 for the vscode plugin, same issue

### Which package is the issue about?

svelte-language-server

### Additional Information, eg. Screenshots

Can work around it with a hack to use require in `svelte.config.js`

```js
import { createRequire } from 'module';
const require = createRequire(import.meta.url);

// eslint-disable-next-line @typescript-eslint/no-var-requires
const pkg = require('./package.json');
```

and language server does not have any problems.

**Comment by jasonlyu123** [comment]
The language server uses the node.js version bundled with VSCode. It's 16.14.2 in VSCode 1.77.1. And json module is under experimental flags before nodejs 16.15. You can work around it by setting the [svelte.language-server.runtime](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#sveltelanguage-serverruntime) config to your node.js installation path. And be sure you set the WSL config to your WSL installation path.

**Comment by jasonlyu123** [comment]
Fixed in VSCode 1.78. The bundled version is now node.js 16.17.1
